src/Sarif/ExtensionMethods.cs

```diff
@@ -401,6 +404,39 @@ public static string GetMessageText(this Result result, ReportingDescriptor rule
             return text;
         }
 
+        public static FailureLevel? GetEffectiveLevel(this Result result, ReportingDescriptor rule)
```

Comment:
>public static [](http://example.com/codeflow?start=8&length=13)

do we need to make this public? This will probably confuse users.

Reply:
agree, changed to internal